### ISSUE TITLE
Create link without creating document (a new .md)

### ISSUE THREAD

**Issue by geekscrapy** [tchayen/markdown-links#60]
I'd love to be able to have some links to an idea which doesn't need a document to be created for it to appear on the graph. This would allow quick thoughts and links to be jotted and the effect would show a graph of overlap of ideas rather than documents. This would enable shorthand brainstorming.

I propose that if even if a link `[[new_car]]` is created, it would appear on the graph even if a file does not exist for it. If it is clicked, in either the graph or in the document, then a file is created.

**Comment by nixsee** [comment]
I wouldn't want this, so if it is implemented, please make it a setting that can be toggled.

**Comment by geekscrapy** [comment]
Can I ask why not? If you want the link to go to a file, you just create the file like you do now and link to it.
Alternatively, if you don't want the link, you just don't make a markdown link in the document?

Question is, do you have lots of markdown links in your documents without the associated files? If you do, then I see that this would break your graph, but if not, I don't see how it changes anything?

**Comment by nixsee** [comment]
> Question is, do you have lots of markdown links in your documents without the associated files?

Yes. 

I don't see why you'd care however - if this feature is implemented, my suggestion to have a setting pleases all parties.

**Comment by geekscrapy** [comment]
I was only interested in your use-case, and to see how another member of the community takes notes... Sorry for the intrusion! 😀

**Comment by nixsee** [comment]
Sorry - I see now that my comment comes across as unnecessarily defensive. Anyway, hopefully this will get implemented for you and others!

**Comment by nixsee** [comment]
This could also have broader implications as well - different YAML tags offering different styling. We've [had a discussion regarding "Parenting" within foam](https://github.com/foambubble/rfcs/pull/5), and we seem to have circled in on just such a feature to create link context/aliases/etc... Not sure which would have to come first - link support for tags or the tags.

**Comment by geekscrapy** [comment]
I love your idea regarding different line styles to indicate if it's an actual file!

My use case is that I have a notes about computers within a network. I work in IT security and will create notes regarding analysis tasks I undertake and any findings. These notes have references to computer names from the network. I'd like to be able to see any overlap of computers between analysis tasks - any computer name that shows up in a graph with a lot of connections would warrant further inspection. But as it is unless there is something specific about a computer I want to note, I don't want to be creating a new file for each computer name...

TLDR; it helps me prioritize my analysis by showing highly connected notes (without creating many files)

**Comment by nixsee** [comment]
Interesting - surely you've already considered (or even implemented) something like this, but it seems like a good job for a spreadsheet with some conditional formatting and filters, or a database with similar functionality. Or perhaps those are too rigid and this is just an easy to use "graph database". 

Anyway, seems like the ideas are out there now - hopefully the various powers that be will bring them into fruition!

**Comment by geekscrapy** [comment]
Yea, I already have these kind of tools in place - this is for smaller scale linking that is made when taking freehand notes during an investigation, nothing structured - these notes may end up in a textual report of some kind. It'd provide me with a very quick and easy way to see shared links and ideas between notes without having a whole backend database to deal with, especially when it'll only be a handful of links, but it then again it could also be 100s of links!

**Comment by nixsee** [comment]
Here is a sneak-peak of an upcoming Filters feature in Obsidian. Green are tags, Red are uncreated notes, blue are attachments

![image](https://user-images.githubusercontent.com/24255237/93381976-accc5100-f81e-11ea-8bc2-88658df614f4.png)

**Comment by tchayen** [comment]
Hi! Sorry for being late to the discussion. I was thinking about displaying links at the very beginning and always considered it something for an optional setting.

But as @nixsee pointed, maybe a whole filter system would be better.

If anyone is interested in implementing that, I'd definitely accept a PR with it with a default `false` for not displaying it.
